content/arweave.ttl

```diff
@@ -3,5 +3,14 @@
 :arweave
     skos:prefLabel "arweave"@en;
     skos:definition '''
-      A decentralized (layer-zero) storage network that provides replication of data over hundreds of Arweave nodes and is funded by the Ar token.
-    '''@en .
+      A decentralized (layer-zero) storage network that provides
+      replication of data over hundreds of Arweave nodes and is
+      funded by the Ar token.
+
+      The network provides an innovative, one-time endowment
+      model that claims to fund the storage of a minimum 40
+      copies of each user file for at least 200 years.
```

Comment:
Propose we remove 'innovative` (marketing jargon)

I am not sure the claim of 200 years helps us either. What are we relying on in Orcfax? probably that it performs its function for now, and maybe for the next 3-5 years, but this should constantly be re-evaluated. 

NB. might be worth reflecting on in the storage retrospective: https://github.com/orcfax/issues/issues/160